npm/tailcall/scripts/generate-packages.js

```diff
@@ -0,0 +1,66 @@
+import * as fs from "node:fs"
+import {dirname, resolve} from "node:path"
+import {fileURLToPath} from "node:url"
+
+const APP_VERSION = process.env.APP_VERSION
+
+const MANIFEST_PATH = resolve(fileURLToPath(import.meta.url), "../..", "package.json")
+
+const rootManifest = JSON.parse(fs.readFileSync(MANIFEST_PATH).toString("utf-8"))
+
+function generatePackages() {
+  const __filename = fileURLToPath(import.meta.url)
+  const __dirname = dirname(__filename)
+
+  let target
+
+  try {
+    target = resolve(__dirname, "../../../dist")
+  } catch {
```

Comment:
if I'm not mistaken the `path.resolve` doesn't check the built path for existence and it just concatenates string in right way. So I think catch error won't happen or at least will have misleading error message

Reply:
Yes, we can remove the `try/catch`.